### PR TITLE
Modifications to OpenApiSpecs for  Adding a url InitClass

### PR DESCRIPTION
**Pull request by paimaam**

### CODE REVIEW

File: src/Kmd.Logic.Sms.Client/Sms/Models/SendSmsRequest.cs [hunk 1]
```diff
@@ -93,24 +93,6 @@ public virtual void Validate()
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "Body");
             }
-            if (ToPhoneNumber != null)
```
Comment (adamchester) [inline src/Kmd.Logic.Sms.Client/Sms/Models/SendSmsRequest.cs]: How did this block of changes happen? Did we use a different version of autorest that stopped doing this validation?

File: src/Kmd.Logic.Sms.Client/Sms/Models/SendTestSmsRequest.cs [hunk 2]
```diff
@@ -67,24 +67,6 @@ public virtual void Validate()
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "Body");
             }
-            if (ToPhoneNumber != null)
```
Comment (adamchester) [inline src/Kmd.Logic.Sms.Client/Sms/Models/SendTestSmsRequest.cs]: How did this block of changes happen? Did we use a different version of autorest that stopped doing this validation?

Comment (paimaam) [reply src/Kmd.Logic.Sms.Client/Sms/Models/SendTestSmsRequest.cs]: Yes, had generated these using beta version  hence caused this,
fixing this using stable version

File: src/Kmd.Logic.Sms.Client/Sms/Models/SendSmsRequest.cs (hunk 1 above)
Comment (paimaam) [reply src/Kmd.Logic.Sms.Client/Sms/Models/SendSmsRequest.cs]: Yes, had generated these using beta version hence caused this,
fixing this using stable version

**Pull request merged**